app/views/locations/show.html.erb

```diff
@@ -1,3 +1,6 @@
 <div id="app">
+<<<<<<< HEAD
```

Comment:
Please delete these git merge artefacts -- carefully review your changes before making a commit, and again before sending out a PR!